libraries/math/src/processor.rs

```diff
@@ -33,5 +33,34 @@ pub fn process_instruction(
             msg!("{}", result);
             Ok(())
         }
+        MathInstruction::U64Multiply {
+            multiplicand,
+            multiplier,
+        } => {
+            msg!("Calculating U64 Multiply");
+            let result = multiplicand + multiplier;
```

Comment:
Post-merge review! Should this be `*`?  Also, now I know the word `multiplicand`